cluster/scoper.go

```diff
@@ -0,0 +1,29 @@
+package cluster
```

Comment:
I don't think this interface is a concern outside the kubernetes package -- namespace scoping is implicit in the Manifests implementation. Granted, resource IDs include a namespace, so there is already some muddling.

Reply:
see my answer above